fabfile.py

```diff
@@ -183,6 +183,50 @@ def update_vendor_config():
         system("cd %s ; git pull" % config_file_dir)
 
 
+@task
+def docker_fabric_task():
```

Comment:
bad task name. Please rename so we know exactly what this task does. Example : docker_piwik_database_backup